### DISCUSSION TITLE
How to make the value of color bar show that of defined in table?

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#944] [Q&A] [answered]
Hi Marco,
I use build_lut function to define a colormap bar. How can I make the value beside the color bar show the value defined in lookup table?
Thank you very much!
Here is the definition of lut:
```
      lut = build_lut([(0,(150,0,0)),
                      (0.35,(255,0,0)),
                      (0.5,(255,150,0)),
                      (0.6,(255,200,0)),
                      (0.75,(255,250,0)),
                      (1,(100,255,0)),
                      (1.5,(0,200,0)),
                      (2,(0,150,0)),
                      (3,(0,200,200)),
                      (4,(0,150,255)),
                      (5,(0,85,255)),
                      (6,(0,0,255)),],
                      vmin=0,vmax=7,below_color=(150,0,0),above_color=(0,0,255),nan_color=(0,0,255),below_alpha=0.68,above_alpha=0.68,nan_alpha=0.18,interpolate=False)

```
![image](https://github.com/marcomusy/vedo/assets/116484589/7e987480-7cfb-4c7c-bdfe-6d84237ff675)

**Comment by marcomusy** [comment, marked as answer]
Hi, At the moment this is not possible in the 2D version of the scalarbar, 
It should be doable in the 3D version - check [this example](https://github.com/marcomusy/vedo/blob/master/examples/volumetric/earth_model.py).

**Reply by ZiguoAtGitHub** [reply]
OK. Thanks.
